index.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Umoru's portfolio</title>
+  <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@300;500;600;700;800&family=Roboto:wght@300;400;500&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.0/css/all.min.css"
+      integrity="sha512-xh6O/CkQoPOWDdYTDqeRdPCVd1SpvCA9XXcUnZS2FmJNp1coAFzvtCN9BmamE+4aHK8yyUHUSCcJHgXloTyT2A=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+    <link rel="stylesheet" href="style.css" />
+</head>
+<body>
+  <div class="container">
+    <header class="header">
+      <nav class="navbar">
+        <a class="logo" href="#">My Logo</a>
+        <div class="hamburger">
+        <img src="./images/outline_menu.png" alt="">
+        </div>
+      </nav>
+  </header>
+
+  <section class="mheadline" id="portfolio-id">
+    <div class="text-container">
+      <h1>
+        I'm Umoru<br />
+        Glad to see you!
+      </h1>
+      <p>
+        I'm a software developer! I can help you build a product , feature or
+        website Look through some of my work and experience! If you like what
+        you see and have a project you need coded, don't hestiate to contact
+        me.
+      </p>
+      </div>
+      <div class="socials">
+        <h6>LET'S CONNECT</h6>
```

Comment:
- [ ] Please 🙏 use an `<a>` tag for the link, Apart from being semantically correct, the `<a>` tag will let you connect to other sections of the website, using `href` 😄 .